team_test.go

```diff
@@ -69,14 +69,359 @@ func TestGetTeamWithAlias(t *testing.T) {
+              "aliases": [
+                "security"
+              ],
+              "contacts": [],
+              "htmlUrl": "https://app.opslevel.com/teams/security",
+              "id": "Z2lkOi8vb3BzbGV2ZWwvVGVhbS84NTE",
+              "manager": {{ template "user_1" }},
+              "members": {
+                "nodes": [
+                  	{{ template "user_4" }},
+					{{ template "user_5" }},
+                    {{ template "user_6" }}
+                ],
+                "pageInfo": {
+                  "hasNextPage": false,
+                  "hasPreviousPage": false,
+                  "startCursor": "MQ",
+                  "endCursor": "Mw"
+                }
+              },
+              "name": "Security",
+              "responsibilities": null
+            },
+            {
+              "alias": "vps",
+              "aliases": [
+                "vps"
+              ],
+              "contacts": [],
+              "htmlUrl": "https://app.opslevel.com/teams/vps",
+              "id": "Z2lkOi8vb3BzbGV2ZWwvVGVhbS84NTI",
+              "manager": {{ template "user_1" }},
+              "members": {
+                "nodes": [
+                  	{{ template "user_4" }},
+					{{ template "user_5" }},
+                    {{ template "user_6" }}
+                ],
+                "pageInfo": {
+                  "hasNextPage": false,
+                  "hasPreviousPage": false,
+                  "startCursor": "MQ",
+                  "endCursor": "Mw"
+                }
+              },
+              "name": "VPs",
+              "responsibilities": null
+            }
+          ],
+{{ template "pagination_second_pageInfo_response" }},
+			"totalCount": 2
+        }
+      }
+    }
+  }`},
+	}
+	client := APaginatedTestClient(t, "team/list", requests...)
+	// Act
+	response, err := client.ListTeams(nil)
+	result := response.Nodes
+	// Assert
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, 5, response.TotalCount)
+	autopilot.Equals(t, "devops", result[0].Alias)
+	autopilot.Equals(t, "developers", result[1].Alias)
+	autopilot.Equals(t, "security", result[3].Alias)
+	autopilot.Equals(t, "vps", result[4].Alias)
+	autopilot.Equals(t, "Own Infra & Tools.", result[0].Responsibilities)
+}
+
+func TestListTeamsWithManager(t *testing.T) {
+	// Arrange
+	requests := []TestRequest{
+		{`{
+    "query": "query TeamList($after:String!$email:String!$first:Int!){account{teams(managerEmail: $email, after: $after, first: $first){nodes{alias,id,aliases,contacts{address,displayName,id,type},group{alias,id},htmlUrl,manager{id,email,htmlUrl,name,role},members{nodes{id,email,htmlUrl,name,role},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount},name,responsibilities,tags{nodes{id,key,value},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}}}",
+	"variables": {
+		"after": "",
+		"first": 100,
+		"email": "kyle@opslevel.com"
```

Comment:
Had to do this unless we want to change the template for this everywhere

Reply:
ahh i see now.  Ya thats fine - kinda the whole point with optionally having templates.  It allows for 1 off's like this